pkg/internal/storage/database/models/output_baskets.go

```diff
@@ -11,11 +12,23 @@ type OutputBasket struct {
 	CreatedAt time.Time
 	UpdatedAt time.Time
 	DeletedAt gorm.DeletedAt `gorm:"index"`
+	Num       int
+
+	Name   string `gorm:"primaryKey;type:varchar(300)"`
+	UserID int    `gorm:"primaryKey"`
```

Comment:
```suggestion
	Name   string `gorm:"primaryKey;type:varchar(300);not null"`
        UserID int    `gorm:"primaryKey;not null"`
```
wonder if we should explictly specify here `not null` WDYT ?